packages/components/src/internal/components/security/APIWrapper.ts

```diff
@@ -156,6 +181,21 @@ export class ServerSecurityAPIWrapper implements SecurityAPIWrapper {
         });
     };
 
+    getDeletionSummaries = (): Promise<Summary[]> => {
+        return new Promise((resolve, reject) => {
+            Ajax.request({
+                url: buildURL('core', 'getModuleSummary.api'),
+                method: 'GET',
+                success: Utils.getCallbackWrapper(response => {
+                    const summaries = response?.moduleSummary;
```

Comment:
Why the defensive check for `response?`? Wouldn't this be an error? Either way this defensive check will be undone by the next line calling `summaries.sort()` if summaries is undefined. I would recommend not checking here.

```js
const { moduleSummary } = response;
moduleSummary.sort(naturalSortByProperty('noun'));
resolve(moduleSummary);
```